distributed/tests/test_scheduler.py

```diff
@@ -194,6 +194,39 @@ def test_remove_worker_from_scheduler(s, a, b):
     s.validate_state()
 
 
+@gen_cluster(config={"distributed.scheduler.events-cleanup-delay": 0.01})
```

Comment:
Slight preference for `"10 ms"` here and below to be explicit about times and units.